_topics/en/transaction-pinning.md

```diff
@@ -71,10 +71,10 @@ Some of the limits that can be abused to enable transaction pinning
 include:
 
 - **[BIP125][] RBF rule #3** requires a replacement transaction
-  pay a higher absolute fee (not just feerate) than the transaction
-  being replaced and any of its children.  This can allow an attacker
-  to attach a large and low-feerate transaction to the transaction
-  they want to pin, forcing any fee bump to pay for the
+  pay a higher absolute fee (not just feerate) than the total fees paid
```

Comment:
Change looks good to me, but I might even prefer:

```suggestion
  pay a higher absolute fee (not just feerate) than the sum of fees paid
```

Reply:
Agreed, force pushed with your suggestion.